lessons/en/chapter_10.yaml

```
@@ -1,10 +1,149 @@
-- title: Chapter 10 - The End
+- title: Chapter 10 - I/O
   content_markdown: >
-    It's been a joy to have you on the Tour of Rust. Ferris and the Tour of Rust
-    team sincerely hope you enjoy the journey ahead! If you 
+    In computing, `I/O` is an abbreviation for `Input/Output` operation.
+  
+    The `input` is that the computer and tha algorithm receive and
+    the `output` represents the result generated based on the `input`.
+  
+    Thing about `I/O` as a stream of information
+  
+    A compute system without output is nearly useless.
+    
+    It will always run the same code on the same data and, thus, produce the same result.
 
-    have felt comfortable this far, we strongly recommend diving deeper with
-    these resources:
+- title: Do it locally
+  content_markdown: >
+    In this chapter, the Playground will be just a code support for you :(.
+
+    Since most of the `I/O` programs are designed to compile on a local machine
+    (yours :) ), consider setting up a Rust environment on your personal computer and
+    familiarise yourself with the terminal.
+
+    Also, consider using a `IDE`, such as `VS Code` or `RustRover`
+    and familiarise yourself with the [terminal](https://www.youtube.com/watch?v=lZ7Kix9bjPI).
+
+- title: Standard Input (stdin)
+  content_markdown: >
+    `Standard Input` refers to the data provided by the user for the algorithm to process.
+
+    
+    Thus, the `input` represents what a program is being given.
+    Mostly, in terms of `input`, you'll work with `String` and `files`.
+
+    
+    The Rust library `std::io` has the necessary components to interact with `I/O`
+    channels, such as the keyboard or any other input source.
+  code: >-
+    https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=use+std%3A%3Aio%3B%0A%0Afn+main%28%29+%7B%0A++++let+mut+input+%3D+String%3A%3Anew%28%29%3B%0A%0A++++%2F%2F+the+read+will+be+stopped+by+%60%5Cn%60+character%0A++++if+let+Ok%28_%29+%3D+io%3A%3Astdin%28%29.read_line%28%26mut+input%29+%7B%0A++++++++println%21%28%22Input+text+%3A+%7B%7D%22%2C+input%29%3B%0A++++%7D%0A%7D%0A
+
+- title: Standard Output (stdout)
+  content_markdown: >
+    Remember the first lesson? Can you notice something relevant to `I/O`?
+
+    Of course that `println!` is does an output operation,
```

Idea expressed unclearly:
```suggestion
    Of course that what `println!` does is an output operation,
```